### ISSUE TITLE
Incorrect date format when creating an RSS feed

### ISSUE THREAD

**Issue by bramley** [fguillot/picoFeed#176]
When creating an RSS feed the publication date is incorrectly formatted with a 2-digit year instead of a 4-digit year.

    <pubDate>Mon, 27 Jul 15 15:13:36 +0100</pubDate>
See http://cyber.law.harvard.edu/rss/rss.html#ltpubdategtSubelementOfLtitemgt for the field specification.

The change required is to use the php constant DATE_RSS instead of DATE_RFC822 in method addPubDate() in PicoFeed/Syndication/Rss20.php
```
    public function addPubDate(DomElement $xml, $value = 0)
    {
        $xml->appendChild($this->dom->createElement(
            'pubDate',
            date(DATE_RFC822, $value ?: time())
        ));
    }
```

**Comment by mkresin** [comment]
Confirmed!

Unfortunately, the RSS 2.0 specification isn't as good as it should be. 

Your link to the item pubDate defines the pubDate as a [RFC 822](http://www.w3.org/Protocols/rfc822/#z28) date (two digit year), but provides a four digit example.

The description of the _optional_ [channel pubDate](http://cyber.law.harvard.edu/rss/rss.html#optionalChannelElements) redefines _all_ dates:

> All date-times in RSS conform to the Date and Time Specification of RFC 822, with the exception that the year may be expressed with two characters or four characters (four preferred).

@bramley Would you please send a pull request.

**Comment by bramley** [comment]
Fixed by #177
